TACC/Core-Portal
# Task/FP-1150: 150 max character limit for shared workspace title

## Overview: ##
Change character limit for shared workspace titles to 150.

## Related Jira tickets: ##

* [FP-1150](https://jira.tacc.utexas.edu/browse/FP-1150
## Summary of Changes: ##
Changed `Yup` validation for max length and warning message to 150 characters.

## Testing Steps: ##
1. Ensure that max character limit is 150 in the **Edit Description** modal and **Create Shared Workspace** modal.

## UI Photos:

## Notes: ##
I saw that the Jira ticket suggested the use of a separate HTML solution (`max_length` on `input`). 
But I wasn't sure, so I just changed the `Yup` validation settings for now.
